tailcall-fixtures/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+use std::path::{Path, PathBuf};
+
+pub fn get_fixture_path(path: impl AsRef<Path>) -> PathBuf {
```

Comment:
```suggestion
pub fn get_proto(path: impl AsRef<Path>) -> PathBuf {
```
Path should be relative to `grpc/proto`. So that we can just simply use `tailcall_fixture::get_proto("news.proto")`.

Reply:
Let's also add some documentation about this function.

Reply:
On second thoughts, can we create a developer experience that looks like:

```rust
let news = FIXTURES::proto::news; 
let cycle = FIXTURES::proto::nested0;
```

Reply:
implemented something like this